src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![feature(termination_trait_lib)]
+#![feature(inner_deref)]
```

Comment:
There is no need for the feature, you can just copy-paste from std (see https://github.com/rust-lang/rust/pull/50267/files), but as we already use nightly it's probably ok...